src/current/v25.4/secure-cockroachdb-operator.md

```diff
@@ -18,9 +293,13 @@ You may need to rotate the node, client, or CA certificates in the following sce
 - The key (for a node, client, or CA) is compromised.
 - You need to modify the contents of a certificate, for example, to add another DNS name or the IP address of a load balancer through which a node can be reached. In this case, you would need to rotate only the node certificates.
 
+Self-signed certificates like those created by the [default TLS configuration](#all-self-signed-certificates) do not auto-rotate. TODO: Give example of how to rotate self-signed (not signed with cockroach cert) certificates
```

Comment:
Need to better understand how to instruct customers in this case, and which configs use `cockroach cert` and would therefore be covered by the example below.